ground/src/main/java/com/google/android/ground/ui/datacollection/DataSubmissionConfirmationDialog.kt

```diff
@@ -52,41 +60,78 @@ fun DataSubmissionConfirmationDialog(onDismiss: () -> Unit) {
-          contentDescription = stringResource(R.string.data_submitted_image),
-          contentScale = ContentScale.Fit,
-        )
-
-        Spacer(modifier = Modifier.height(12.dp))
-
-        Text(
-          text = stringResource(R.string.data_collection_complete),
-          style = MaterialTheme.typography.titleLarge,
-          textAlign = TextAlign.Center,
-        )
-
-        Spacer(modifier = Modifier.height(8.dp))
+      val configuration = LocalConfiguration.current
+      if (configuration.orientation == Configuration.ORIENTATION_LANDSCAPE) {
+        Column {
+          Row(verticalAlignment = Alignment.CenterVertically) {
+            DataCollectionThumbnail(modifier = Modifier.weight(1f))
+            Column(
+              modifier = Modifier.padding(start = 16.dp, end = 24.dp, top = 16.dp).weight(2f),
+              verticalArrangement = Arrangement.Center,
+            ) {
+              DetailColumn()
+            }
+          }
+          CloseButton(modifier = Modifier.padding(bottom = 24.dp, end = 24.dp), onDismiss)
+        }
+      } else {
+        Column(modifier = Modifier.padding(horizontal = 24.dp, vertical = 16.dp)) {
+          DataCollectionThumbnail()
+          DetailColumn()
+          Spacer(modifier = Modifier.height(12.dp))
+          CloseButton(onDismiss = onDismiss)
+        }
+      }
+    }
+  }
+}
 
-        Text(
-          text = stringResource(R.string.data_collection_complete_details),
-          style = MaterialTheme.typography.bodyMedium,
-          textAlign = TextAlign.Center,
-        )
+@Composable
+private fun DataCollectionThumbnail(modifier: Modifier = Modifier) {
+  Image(
+    modifier = modifier,
+    painter = painterResource(id = R.drawable.data_submitted),
+    contentDescription = stringResource(R.string.data_submitted_image),
+    contentScale = ContentScale.Fit,
+  )
+}
 
-        Spacer(modifier = Modifier.height(12.dp))
+@Composable
+private fun ColumnScope.DetailColumn() {
+  Text(
+    text = stringResource(R.string.data_collection_complete),
+    style = MaterialTheme.typography.titleLarge,
+    color = md_theme_light_onBackground,
+    lineHeight = 28.sp,
+  )
+  Spacer(modifier = Modifier.height(8.dp))
+  Text(
+    text = stringResource(R.string.data_collection_complete_details),
+    fontSize = 16.sp,
+    lineHeight = 24.sp,
+    color = MaterialTheme.colorScheme.onSurfaceVariant,
+    fontFamily = manropeFont,
+  )
+}
 
-        OutlinedButton(onClick = { onDismiss() }) { Text(stringResource(id = R.string.close)) }
-      }
-    }
+@Composable
+private fun ColumnScope.CloseButton(modifier: Modifier = Modifier, onDismiss: () -> Unit) {
+  OutlinedButton(
+    modifier = modifier.align(Alignment.End),
+    border = BorderStroke(width = 1.dp, color = MaterialTheme.colorScheme.outline),
+    onClick = { onDismiss() },
+  ) {
+    Text(
+      stringResource(id = R.string.close),
+      fontSize = 14.sp,
+      lineHeight = 20.sp,
+      fontFamily = manropeFont,
```

Comment:
Can we use `fontFamily = FontFamily(Font(R.font.text_500)),` instead?

Reply:
fixed